static/src/js/pages/KeywordManagerPage/KeywordManagerPage.jsx

```diff
@@ -53,13 +53,23 @@ const KeywordManagerPage = () => {
   const [isTreeLoading, setIsTreeLoading] = useState(false)
   const { kmsHost } = getApplicationConfig()
   const [treeMessage, setTreeMessage] = useState('Select a version and scheme to load the tree')
+  const [reloadTree, setReloadTree] = useState(false)
+  const [selectedKeywordId, setSelectedKeywordId] = useState(null)
+
   const [showPublishModal, setShowPublishModal] = useState(false)
   const [newVersionName, setNewVersionName] = useState('')
   const [publishError, setPublishError] = useState(null)
   const [showKeywordForm, setShowKeywordForm] = useState(false)
   const [showPublishingModal, setShowPublishingModal] = useState(false)
   const [versionSelectorKey, setVersionSelectorKey] = useState(0)
-  const { tokenValue } = useAuthContext()
+  const { tokenValue, user } = useAuthContext()
+  const { uid } = user || {}
+
+  const handleKeywordSave = useCallback((savedKeywordId) => {
+    setReloadTree((prev) => !prev)
```

Comment:
```suggestion
    setReloadTree(!reloadTree)
```
confused why this needs to be arrow func? can't we just pass the inverse of the current state here?

Reply:
Done